src/legacy/ui/public/vis/editors/default/components/agg_group.tsx

```diff
@@ -80,7 +82,15 @@ function DefaultEditorAggGroup({
 
   const [aggsState, setAggsState] = useReducer(aggGroupReducer, group, initAggsState);
 
-  const isGroupValid = Object.values(aggsState).every(item => item.valid);
+  let bucketsError;
```

Comment:
nit: we can use a ternary condition to avoid using `let` here: 
```ts
const bucketsError = lastParentPipelineAggTitle && groupName === AggGroupNames.Buckets && !group.length ? i18n.translate('common.ui.aggTypes.buckets.mustHaveBucketErrorMessage', {
      defaultMessage: 'Add a bucket with "Date Histogram" or "Histogram" aggregation.',
      description: 'Date Histogram and Histogram should not be translated',
    }) : undefined;
```